demo/keras/TUTORIAL_CN.md

```diff
@@ -0,0 +1,90 @@
+# 如何在Keras中使用VisualDL
+
+下面我们演示一下如何在Keras中使用VisualDL，从而可以把Keras的训练过程可视化出来。我们将以Keras用卷积神经网络(CNN, Convolutional Neural Network)来训练
+[MNIST](http://yann.lecun.com/exdb/mnist/) 数据集作为例子。
+
+
+程序的主体来自Keras的官方GitHub [Example](https://github.com/keras-team/keras/blob/master/examples/mnist_cnn.py)。
+
+我们只需要在代码里面创建 VisualDL 的数据采集 loggers
+
+```python
+# create VisualDL logger
+logdir = "/workspace"
+logger = LogWriter(logdir, sync_cycle=100)
+
+# mark the components with 'train' label.
+with logger.mode("train"):
+    # create a scalar component called 'scalars/'
+    scalar_keras_train_loss = logger.scalar(
+        "scalars/scalar_keras_train_loss")
+    image0 = logger.image("images/image0", 1)
```

Comment:
why do we need to prefix with "images/"?  Isn't this graph only going to show up in images section?

Reply:
'images/' is showing that we can group metrics by namespace, which will be implemented in the future.